Assets/MixedRealityToolkit/Graph/GraphConnector.cs

```diff
@@ -0,0 +1,121 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Graph.Internal;
+using System.Threading.Tasks;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Graph
+{
+    /// <summary>
+    /// Unity script that enables access to the Graph.
+    /// </summary>
+    public class GraphConnector : MonoBehaviour
```

Comment:
Does this need to inherit from Mono?

Reply:
The reason why it's inheriting from Mono is to allow user to set the Graph settings through the Unity editor, using the inspector to make it Unity friendly. I will be fine to change if you feel strong about it?

Currently there are only 3 settings (App Id, Access Scopes, Auth Token) and there might be more to add in the future to support more (uncommon) authentication scenarios (e.g. Authority, RedirectUri, Extra Scopes, etc).

Reply:
You can do that with the ScriptableObjects as well. Please revisit my PR against your branch.

Reply:
I don't think this class should be a monobehaviour